src/app/modules/navbar/navbar.component.html

```diff
@@ -161,7 +149,4 @@
       </div>
     </div>
   </div>
-</div>
-
-<app-confirm-email-modal *ngIf="showConfirmEmailModal" (close)="showConfirmEmailModal = false">
```

Comment:
Is it possible to remove the associated import for the confirmation modal?

Reply:
Yes, removed unused module from navbar.